doc/user/content/ops/monitoring.md

```diff
@@ -175,3 +175,76 @@ Configuration parameter | Value
 `prometheus_url`        | `http://<materialized host>/metrics`
 `namespace`             | Your choice
 `metrics`               | `[mz*]` to select all metrics, or a list of specific metrics
+
+## System Logs
+
+Materialize emits logs to the stream specified by the `--log-file` invocation
+parameter, documented at [cli](/cli).
+
+Materialize strives to make it easy to monitor logs, and so we have clear
+standards for each of our log levels. The high-level goal is that materialize
+running well should *never* emit `WARN` or `ERROR` level logs, every log at that
+level should be resolved.
+
+By default materialized logs only `INFO`, `WARN` and `ERROR` logs, but it is
+possible to enable more-verbose logs by setting the `MZ_LOG` environment
```

Comment:
CC @benesch do we want to change this to `MZ_LOG_LEVEL` and expose it as `--log-level`?

Reply:
or also possibly `MZ_LOG_FILTER`/`--log-filter`

Reply:
Yes, I've wanted to do that forever! `MZ_LOG_FILTER`/`--log-filter` sounds great to me.

Reply:
#6317